Analysis/src/BaseEventAnalyzer.cpp

```diff
@@ -129,7 +129,7 @@ EventSubCategory BaseEventAnalyzer::DetermineEventSubCategory(EventInfoBase& eve
     if(event.HasBjetPair()){
         mbb = event.GetHiggsBB().GetMomentum().mass();
         if(category.HasBoostConstraint() && category.IsBoosted()){
-            if(ana_setup.use_svFit){
+            if(ana_setup.use_svFit && event.GetSVFitResults().has_valid_momentum){
```

Comment:
SetCutResult should be called independently whatever SVfit moment is valid or not (in case if it is not valid the result will be false, of course)